test/pummel/test-heapsnapshot-near-heap-limit-by-api.js

```diff
@@ -0,0 +1,131 @@
+// Copy from test-heapsnapshot-near-heap-limit.js
+'use strict';
+
+const common = require('../common');
+
+if (common.isPi) {
+  common.skip('Too slow for Raspberry Pi devices');
+}
+
+const tmpdir = require('../common/tmpdir');
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+const fixtures = require('../common/fixtures');
+const fs = require('fs');
+const v8 = require('v8');
+
+if (process.env.isChild === '1') {
```

Comment:
As the script is complicated enough it might be better to simply create a fixtures script for it.

Reply:
Done !